pkg/user/repository.go

```diff
@@ -32,6 +32,17 @@ func (r repository) create(u *model.User) error {
 	return err
 }
 
+func (r repository) findUsers() ([]*model.User, error) {
```

Comment:
I am not sure if I would call it `findUsers` as we are getting all users. I would use `find` when we have some property/attribute we use to find a record. Not sure if `getAll`/`getAllUsers` would be better 🤔

Reply:
I think that both `find` and `get` can both entail finding/getting something by a property, so I went with `find` as it's more commonly used in the code base (actually, there's hardly any `getSomething` methods). I.e. I'm slightly in favour of `findAll`. What do you think?

Reply:
> About casing - I'm not sure I'm following. Are we talking about `clusterConfiguration` compared to the rest which are `CapitalCase`? But in any case, I do agree that all responses should be consistent in terms of naming and casing. +1

It's not just `clusterConfiguration` which is lower case but that aside. Last time I investigated I didn't find a way to globally define this. So we have to add struct tags to every property we return.

Reply:
find and get have slightly different meaning to me but I might be wrong 😁 find: search for something that might not be there, get something I know exists. `findAll` does make it more clear at least. I'll let you decide on `findAll/getAll` 😋

Reply:
@radnov, @teleivo - I've been using findAll elsewhere. Let's stick with that or change it globally :pray: